movebucket_test.go

```diff
@@ -0,0 +1,301 @@
+package bbolt_test
+
+import (
+	"bytes"
+	"math/rand"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	crand "crypto/rand"
```

Comment:
Please also sort the import items
```suggestion
	crand "crypto/rand"
```